src/install/dracut-install.c

```diff
@@ -500,11 +500,11 @@ static int resolve_deps(const char *src)
         if (!fullsrcpath)
                 return 0;
 
-        buf = malloc(LINE_MAX);
+        buf = calloc(LINE_MAX, sizeof(char));
```

Comment:
The C standard specifies that sizeof(char) == 1. Just use 1 like everyone else.

Reply:
If you insist I will do it but IMO this is more readable. Anyway this will get resolved at compile time, so it should not be a problem. I saw that there is a malloc0 macro in util.h, I will use it instead. Thanks.